package/tests/test_cp/test_openstack/test_common/test_deploy_data_holder.py

```diff
@@ -0,0 +1,87 @@
+from unittest import TestCase
+
+import mock
+
+from cloudshell.cp.openstack.common.deploy_data_holder import DeployDataHolder
+
+
+class TestDeployDataHolder(TestCase):
+
+    def setUp(self):
+        super(TestDeployDataHolder, self).setUp()
+        self.tested_class = DeployDataHolder
+
+    def test_create_from_params(self):
```

Comment:
This is not the problem with the way test-case is written per se, but I believe `create_from_params` is a side effect of original code. So I will accept this change, but later on delete, both the original code and this test case.